### PR TITLE
adopting the current setting to our users configuration

### PR DESCRIPTION
**Pull request by teamagam1**
adopting the current setting to our users configuration

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/47" target="_blank">ReviewNinja

File: app/src/main/res/xml/pref_general.xml [hunk 1]
```diff
@@ -1,33 +1,12 @@
...
-        android:title="@string/pref_title_social_recommendations" />
-
-    <!-- NOTE: EditTextPreference accepts EditText attributes. -->
-    <!-- NOTE: EditTextPreference's summary should be set to its value by the activity code. -->
     <EditTextPreference
         android:capitalize="words"
         android:defaultValue="@string/pref_default_display_name"
         android:inputType="textCapWords"
-        android:key="example_text"
+        android:key="user_name_text"
```
Comment (BarAgam) [inline app/src/main/res/xml/pref_general.xml]: extract to string resource !fix

File: app/src/main/res/xml/pref_gps.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <SwitchPreference
+        android:defaultValue="true"
+        android:key="notifications_gps"
+        android:title="@string/pref_title_gps_availability" />
+
+    <ListPreference
+        android:defaultValue="180"
+        android:entries="@array/pref_sync_gps_distance"
+        android:entryValues="@array/pref_sync_gps_distance_values"
+        android:key="gps_distance"
```
Comment (BarAgam) [inline app/src/main/res/xml/pref_gps.xml]: extract to string resouce !fix

Comment (BarAgam) [inline app/src/main/res/xml/pref_gps.xml]: extract to string resouce !fix

File: app/src/main/res/xml/pref_mesages.xml [hunk 3]
```diff
@@ -7,15 +7,9 @@
         android:defaultValue="180"
         android:entries="@array/pref_sync_frequency_titles"
         android:entryValues="@array/pref_sync_frequency_values"
-        android:key="sync_frequency"
+        android:key="sync_messages_frequency"
```
Comment (BarAgam) [inline app/src/main/res/xml/pref_mesages.xml]: extract to string resource !fix

File: app/src/main/res/xml/pref_gps.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <!-- A 'parent' preference, which enables/disables child preferences (below)
+         when checked/unchecked. -->
+    <SwitchPreference
+        android:defaultValue="true"
+        android:key="notifications_gps"
```
Comment (BarAgam) [inline app/src/main/res/xml/pref_gps.xml]: extract to string resource !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/view/settings/SettingsActivity.java [hunk 5]
```diff
@@ -182,8 +153,7 @@ public void onCreate(Bundle savedInstanceState) {
             // to their values. When their values change, their summaries are
             // updated to reflect the new value, per the Android Design
             // guidelines.
-            bindPreferenceSummaryToValue(findPreference("example_text"));
-            bindPreferenceSummaryToValue(findPreference("example_list"));
+            bindPreferenceSummaryToValue(findPreference("user_name_text"));
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/settings/SettingsActivity.java]: user string resource !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/view/settings/SettingsActivity.java [hunk 6]
```diff
@@ -202,18 +172,18 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
-            addPreferencesFromResource(R.xml.pref_notification);
+            addPreferencesFromResource(R.xml.pref_gps);
             setHasOptionsMenu(true);
 
             // Bind the summaries of EditText/List/Dialog/Ringtone preferences
             // to their values. When their values change, their summaries are
             // updated to reflect the new value, per the Android Design
             // guidelines.
-            bindPreferenceSummaryToValue(findPreference("notifications_new_message_ringtone"));
+            bindPreferenceSummaryToValue(findPreference("gps_distance"));
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/settings/SettingsActivity.java]: use string resource !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/view/settings/SettingsActivity.java [hunk 7]
```diff
@@ -232,18 +202,15 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+            addPreferencesFromResource(R.xml.pref_mesages);
             setHasOptionsMenu(true);
 
-            // Bind the summaries of EditText/List/Dialog/Ringtone preferences
-            // to their values. When their values change, their summaries are
-            // updated to reflect the new value, per the Android Design
-            // guidelines.
-            bindPreferenceSummaryToValue(findPreference("sync_frequency"));
+
+            bindPreferenceSummaryToValue(findPreference("sync_messages_frequency"));
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/settings/SettingsActivity.java]: use string resource !fix

**Pull request merged**